### PR TITLE
feat(aws-lambda): Add docs on using Sentry as a Container Image

### PR DESCRIPTION
**Pull request by ahmedetefy**
This PR adds docs for both Python and Node on how to use Sentry's AWS Layer as a container image on AWS

### CODE REVIEW

File: src/platforms/node/guides/aws-lambda/container-image/index.mdx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Replace **VERSION** with a specific version number (for example, 28). You can see a complete list of available versions in Sentry's [Amazon ECR repository](https://gallery.ecr.aws/sentry/sentry-node-serverless-sdk).
+
+## Function Configuration {#function-configuration}
+
+Set the following environment variables in AWS:
+- Set `NODE_OPTIONS` to `-r @sentry/serverless/dist/awslambda-auto`
+- Set `SENTRY_DSN` to your Sentry's DSN
+- Set `SENTRY_TRACES_SAMPLE_RATE` to your preferred [sampling rate](/platforms/python/configuration/sampling/#sampling-transaction-events) for transactions
+
+Alternatively, you can also set the environment vairables in the Dockerfile:
```
Comment (imatwawana) [inline src/platforms/node/guides/aws-lambda/container-image/index.mdx:21]:
```suggestion
Alternatively, you can also set the environment variables in the Dockerfile:
```

File: src/platforms/node/guides/aws-lambda/container-image/index.mdx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+description: "Learn how to set up Sentry with a Container Image"
+---
+
+You can also add Sentry to your [Container Image](https://docs.aws.amazon.com/lambda/latest/dg/images-create.html). To import Sentry's Layer Image, add the following to your Dockerfile:
+
+```{tabTitle: Dockerfile}
+COPY --from=public.ecr.aws/sentry/sentry-node-serverless-sdk:<VERSION> /opt/ /opt
+```
+
+Replace **VERSION** with a specific version number (for example, 28). You can see a complete list of available versions in Sentry's [Amazon ECR repository](https://gallery.ecr.aws/sentry/sentry-node-serverless-sdk).
```
Comment (imatwawana) [inline src/platforms/node/guides/aws-lambda/container-image/index.mdx:12]:
```suggestion
Replace `VERSION` with a specific version number (for example, 28). You can see a complete list of available versions in Sentry's [Amazon ECR repository](https://gallery.ecr.aws/sentry/sentry-node-serverless-sdk).
```

File: src/platforms/node/guides/aws-lambda/container-image/index.mdx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+Alternatively, you can also set the environment vairables in the Dockerfile:
+
+```{tabTitle: Dockerfile}
+ENV NODE_OPTIONS="-r @sentry/serverless/dist/awslambda-auto"
+ENV SENTRY_DSN="___PUBLIC_DSN___"
+ENV SENTRY_TRACES_SAMPLE_RATE="1.0"
+```
+
+**NOTE:** The values you set in AWS overrides the value in the Dockerfile if both are set.
```
Comment (imatwawana) [inline src/platforms/node/guides/aws-lambda/container-image/index.mdx:29]:
```suggestion
<Note>

The values you set in AWS override the value in the Dockerfile if both are set.

</Note>
```

File: src/platforms/python/guides/aws-lambda/container-image/index.mdx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+description: "Learn how to set up Sentry with a Container Image"
+---
+
+You can also add Sentry to your [Container Image](https://docs.aws.amazon.com/lambda/latest/dg/images-create.html). To import Sentry's Layer Image, add the following to your Dockerfile:
+
+```{tabTitle: Dockerfile}
+COPY --from=public.ecr.aws/sentry/sentry-python-serverless-sdk:<VERSION> /opt/ /opt
+```
+
+Replace **VERSION** with a specific version number (for example, 6). You can see a complete list of available versions in Sentry's [Amazon ECR repository](https://gallery.ecr.aws/sentry/sentry-python-serverless-sdk).
```
Comment (imatwawana) [inline src/platforms/python/guides/aws-lambda/container-image/index.mdx:12]:
```suggestion
Replace `VERSION` with a specific version number (for example, 6). You can see a complete list of available versions in Sentry's [Amazon ECR repository](https://gallery.ecr.aws/sentry/sentry-python-serverless-sdk).
```

File: src/platforms/python/guides/aws-lambda/container-image/index.mdx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+## Function Configuration {#function-configuration}
+
+Set your image’s CMD value to the Sentry Handler:
+```{tabTitle: Dockerfile}
+CMD ["sentry_sdk.integrations.init_serverless_sdk.sentry_lambda_handler"]
+```
+
+Also, Set the following environment variables in AWS:
```
Comment (imatwawana) [inline src/platforms/python/guides/aws-lambda/container-image/index.mdx:21]:
```suggestion
Also, set the following environment variables in AWS:
```

File: src/platforms/python/guides/aws-lambda/container-image/index.mdx [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+- Set `SENTRY_INITIAL_HANDLER` to the path of your function handler
+- Set `SENTRY_DSN` to your Sentry's DSN
```
Comment (imatwawana) [inline src/platforms/python/guides/aws-lambda/container-image/index.mdx:23]:
```suggestion
- Set `SENTRY_DSN` to your Sentry DSN
```

File: src/platforms/python/guides/aws-lambda/container-image/index.mdx [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+- Set `SENTRY_TRACES_SAMPLE_RATE` to your preferred [sampling rate](/platforms/python/configuration/sampling/#sampling-transaction-events) for transactions
+
+Alternatively, you can also set the environment vairables in the Dockerfile:
```
Comment (imatwawana) [inline src/platforms/python/guides/aws-lambda/container-image/index.mdx:26]:
```suggestion
Alternatively, you can also set the environment variables in the Dockerfile:
```

File: src/platforms/python/guides/aws-lambda/container-image/index.mdx [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+
+```{tabTitle: Dockerfile}
+ENV SENTRY_INITIAL_HANDLER="<PATH_TO_FUNCTION_HANDLER>"
+ENV SENTRY_DSN="___PUBLIC_DSN___"
+ENV SENTRY_TRACES_SAMPLE_RATE="1.0"
+```
+
+**NOTE:** The values you set in AWS overrides the value in the Dockerfile if both are set.
```
Comment (imatwawana) [inline src/platforms/python/guides/aws-lambda/container-image/index.mdx:34]:
```suggestion
<Note>

The values you set in AWS override the value in the Dockerfile if both are set.

</Note>
```

File: src/platforms/python/guides/aws-lambda/index.mdx [hunk 9]
```diff
@@ -23,7 +23,7 @@ Install our Python SDK using `pip`:
 ```bash
 pip install --upgrade sentry-sdk
 ```
-We also support [installing Sentry in Lambda Layer](/platforms/python/guides/aws-lambda/layer/).
+We also support [installing Sentry in Lambda Layer](/platforms/python/guides/aws-lambda/layer/) and [installing Sentry as a Container Image](/platforms/python/guides/aws-lambda/container-image/).
```
Comment (imatwawana) [inline src/platforms/python/guides/aws-lambda/index.mdx:26]: Switching the order of these to match the order in which they're listed in the navigation

```suggestion
We also support [installing Sentry as a Container Image](/platforms/python/guides/aws-lambda/container-image/) and [installing Sentry in Lambda Layer](/platforms/python/guides/aws-lambda/layer/).
```

File: src/platforms/node/guides/aws-lambda/index.mdx [hunk 10]
```diff
@@ -26,7 +26,7 @@ npm install --save @sentry/serverless
 yarn add @sentry/serverless
 ```
 
-We also support [installing Sentry in Lambda Layer](/platforms/node/guides/aws-lambda/layer/).
+We also support [installing Sentry in Lambda Layer](/platforms/node/guides/aws-lambda/layer/) and [installing Sentry as a Container Image](/platforms/node/guides/aws-lambda/container-image/).
```
Comment (imatwawana) [inline src/platforms/node/guides/aws-lambda/index.mdx:29]: Switching the order of these to match the order in which they're listed in the navigation

```suggestion
We also support [installing Sentry as a Container Image](/platforms/node/guides/aws-lambda/container-image/) and [installing Sentry in Lambda Layer](/platforms/node/guides/aws-lambda/layer/).
```

File: src/platforms/node/guides/aws-lambda/container-image/index.mdx [hunk 11]
```diff
@@ -0,0 +1,29 @@
+---
+title: AWS Lambda Container Image (Node)
+description: "Learn how to set up Sentry with a Container Image"
```
Comment (imatwawana) [inline src/platforms/node/guides/aws-lambda/container-image/index.mdx:3]:
```suggestion
description: "Learn how to set up Sentry with a Container Image."
```

File: src/platforms/python/guides/aws-lambda/container-image/index.mdx [hunk 12]
```diff
@@ -0,0 +1,34 @@
+---
+title: AWS Lambda Container Image (Python)
+description: "Learn how to set up Sentry with a Container Image"
```
Comment (imatwawana) [inline src/platforms/python/guides/aws-lambda/container-image/index.mdx:3]:
```suggestion
description: "Learn how to set up Sentry with a Container Image."
```

**Review by imatwawana — APPROVED** [review]
Minor changes. Otherwise LGTM!

**Review by scefali — APPROVED** [review]
@ahmedetefy looks good to me overall, just some small fixes that @imatwawana pointed out

**Comment by ahmedetefy** [comment]
@imatwawana Thanks for the thorough review :)

**Pull request merged**